### PR TITLE
Implemented the publishing to npm through cci master build

### PR DESCRIPTION
**Pull request by rodrigopavezi**
Add circle ci commands
Introduced a new dev library called publish
Changed the entry point js file name to rise-cache.js
Changed version and add other command to the package.json

@donnapep @stulees please review. cheers

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-    version: 5.0.0
\ No newline at end of file
+    version: 5.0.0
+deployment:
+  npm:
+    branch: master
+    commands:
+      # login using environment variables
+      - echo -e "$NPM_USERNAME\n$NPM_PASSWORD\n$NPM_EMAIL" | npm login
+      - npm run 2npm
```
Comment (donnapep) [inline circle.yml]: @rodrigopavezi Do we need a module to do this? We should be able to publish with just `npm publish`.

Comment (rodrigopavezi) [inline circle.yml]: @donnapep this module have some hand stuff with nice error messages. I looked up on the web and it seems to be used a lot. cheers

Comment (donnapep) [inline circle.yml]: @rodrigopavezi OK, sounds good.

**Comment by donnapep** [comment]
👍

**Pull request merged**